modules/form-field/css/lib/form-field.scss

```diff
@@ -70,12 +69,17 @@
 }
 
 .wdc-form-group {
+  padding: 0;
```

Comment:
Can you add a comment that these styles override the default fieldset styles so they don't get accidentally removed again?